py/setup.py

```diff
@@ -239,7 +239,7 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
+            dir_path + "/../bazel-Torch-TensorRT/external/tensorrt/include",
```

Comment:
we already have this in the line 243. Last time I checked with @narendasan , we wanted to keep TRTorch here to support users who didn't clone the repo (after the rebranding). Those users will still have the directory name TRTorch and corresponding bazel dirs names.